hpc-cluster-setup/aws/v3/README.md

```diff
@@ -77,6 +77,25 @@ Upload the built package from `_out` folder to a s3 bucket and update the url in
 
 Note: Add Subnet with Public IP for headnode and Private IP for compute nodes.
 
+Create capacity reservation policy for Instance reservation
```

Comment:
Please mark this as an optional step needed in case user is using reserved instances